client/src/scene/SceneGameArena.ts

```diff
@@ -212,9 +208,9 @@ export class SceneGameArena extends Phaser.Scene {
 
     private updateFalling() {
         if (this.gameState.playerId == null) {
-            return;
-        }
-        if (this.gameState.moveIfCan(Tetromino.fall)) {
+            // is spectator
+            this.gameState.updateLineClearing();
```

Comment:
Is this needed? A spectator isn't moving any tetromino on the fall timer so I don't think there's a need to check for line clears

Reply:
Nice catch, this is actually an unconscious workaround of a bug in spectator handling `playerPlaced` events.